pkgs/by-name/ch/chartdb/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nodejs_24,
+}:
+let
+  name = "chartdb";
+  version = "1.19.0"; # https://github.com/chartdb/chartdb/releases
+in
+buildNpmPackage {
```

Please use `finalAttrs` to avoid `let`/`in`/`rec` drawbacks
```suggestion
buildNpmPackage (finalAttrs: {
```